clang/include/clang/Basic/Attr.td

```diff
@@ -1415,13 +1415,30 @@ def LoopUnrollHint : StmtAttr {
 }
 
 def IntelReqdSubGroupSize: InheritableAttr {
-  let Spellings = [GNU<"intel_reqd_sub_group_size">,
-                   CXX11<"intel", "reqd_sub_group_size">];
+  let Spellings = [
+    GNU<"intel_reqd_sub_group_size">, CXX11<"intel", "reqd_sub_group_size">,
+    CXX11<"intel", "sub_group_size"> // SYCL2020 spelling.
+  ];
   let Args = [ExprArgument<"Value">];
   let Subjects = SubjectList<[Function], ErrorDiag>;
   let Documentation = [IntelReqdSubGroupSizeDocs];
   let LangOpts = [OpenCL, SYCLIsDevice, SilentlyIgnoreSYCLIsHost];
   let SupportsNonconformingLambdaSyntax = 1;
+  let AdditionalMembers = [{
+    bool isSYCL2020Spelling() const {
+      return getSemanticSpelling() == CXX11_intel_sub_group_size;
+    }
+  }];
```

Comment:
```suggestion
  let Accessors = [Accessor<"isSYCL2020Spelling", [CXX11<"intel", "sub_group_size">]>];
```
Sorry for not catching this sooner, but that's what accessors are for.

Reply:
Neat!  I didn't realize this was a thing.